src/assets/stylesheets/extensions/pymdown/_tabbed.scss

```diff
@@ -59,8 +59,26 @@
         font-weight: 700;
 
         // Show code tab content
-        & + .superfences-content {
+        & + .tabbed-content {
           display: block;
+          padding: 0 px2rem(12px);
+
+          // Adjust spacing on last element
+          > :last-child {
+            margin-bottom: px2rem(12px);
+          }
+
+          > {
+            div.highlight,
+            div.codehilite,
+            pre,
+            table.highlighttable,
+            table.codehilitetable {
+              &:last-child:first-child {
```

Comment:
I think we can use `:only-child` here, as supports seems [to be good](https://developer.mozilla.org/de/docs/Web/CSS/:only-child).

Also, can’t we drop `div.higlight` and `div.codehilite` selectors due to Markdown 3.2?

Reply:
:man_facepalming: yup. Don't know why I fixated on doing it like this.

Reply:
... a conversation of two people, both of which have implemented a selector engine 😂

Reply:
I know, that is the most embarrassing part...I implemented `:only-child`...

As far as `div.highlight` and `div.codehilite` is concerned, this is one case where we cannot drop it. For styling code blocks, all we care about is the `<pre><code>` part, but in this case, we want to know if the code block is an only child under the tab, and Pygments still wraps the `<pre><code>` in a `<div>`. It's just `<div><pre><code>` now instead of `<div><pre>`. And since we need to know that the only child `<div>` is a code `<div>`, we need to check the class because we can't just go `div:has(pre)`.

Reply:
You’re right! That’s also why the child selector is absolutely necessary here.

Reply:
... but we could drop the tag parts (`table`, `div`) of the selectors. Probably even more future-proof.

Reply:
Agreed, table and div are unnecessary.